docs/getting-started.md

```diff
@@ -2,28 +2,40 @@
 
 De 'Bevraging Ingeschreven Persoon' Web API is gespecificeerd met behulp van de OpenAPI specifications (OAS).
 
-Een visuele representatie van de specificatie kan worden gegenereerd met [Swagger UI](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/Bevraging-Ingeschreven-Persoon/resolved/openapi.yaml).
+Om de API te gebruiken kun je de volgende stappen doorlopen:
+1. Bekijk de [functionaliteit en specificaties](#Functionaliteit-en-specificaties)
+2. [Implementeer](#Implementeer-de-API) de API
+3. [Probeer en test](#Probeer-en-test) de API
 
-De OAS specificatie van de 'Bevraging Ingeschreven Persoon' Web API is te downloaden van de [Haal-Centraal-BRP-bevragen github repository](https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/master/api-specificatie/Bevraging-Ingeschreven-Persoon/resolved/openapi.yaml).
+## Functionaliteit en specificaties
+Een visuele representatie van de specificatie kan worden gegenereerd met [Swagger UI](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/specificatie/genereervariant/openapi.yaml).
 
-De [functionele documentatie](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/tree/master/features) van de 'Bevraging Ingeschreven Persoon' Web API is ook te vinden in de github repository.
+De [functionele documentatie](../../../features) van de 'Bevraging Ingeschreven Persoon' Web API is ook te vinden in de github repository.
 
-De 'Bevraging Ingeschreven Persoon' Web API is te benaderen via de volgende url: https://www.haalcentraal.nl/haalcentraal/api/brp
+## Implementeer de API
+Je kunt code genereren op basis van de [genereervariant van de specificaties](../specificatie/genereervariant/openapi.yaml).
+We hebben al voor enkele ontwikkelomgevingen [client code](../../../code) gegenereerd.

```

Comment:
../../../code = ../code?

Reply:
dat had ik eerst gedaan. Maar getting-started staat in /blob/.... en een map staat in /tree/... Dan werkt de relatieve link niet